src/stories/Library/Lists/list-reservation/list-reservation.scss

```diff
@@ -111,10 +129,28 @@ $list-reservation-space-desktop: 24px;
   @include breakpoint-s {
     align-items: center;
   }
+
+  // Hide arrow
+  > svg {
+    width: 35px;
+    visibility: hidden;
+    opacity: 0;
+    margin-right: 20px;
+  }
+  // Show arrow on hover
+  &:hover svg {
+    @include breakpoint-s {
+      visibility: visible;
+      opacity: 1;
+      align-self: center;
+      justify-self: end;
+    }
+  }
 }
```

Comment:
Seems like the exact same thing that is done in[ search result items](https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/f3c0e9c4b545e4064fbe7f8f19af4b69234d03df/src/stories/Library/search-result-item/search-result-item.scss#L22-L36).
How about transforming it into a mixin?